drivers/github_release/backoff.go

```diff
@@ -0,0 +1,45 @@
+package template
+
+import (
+	"math/rand"
+	"time"
+)
+
+const (
+	initialRetryInterval = 500 * time.Millisecond
+	maxInterval          = 1 * time.Minute
+	maxElapsedTime       = 15 * time.Minute
+	randomizationFactor  = 0.5
+	multiplier           = 1.5
```

Comment:
默认的时间太长了，之前这个 backoff 用在一个本地长时间的业务上，这个场景应对墙内网络故障，时间应该适当条短一些。应该调整成
```suggestion
	initialRetryInterval = 500 * time.Millisecond
	maxInterval          = 5 * time.Second
	maxElapsedTime       = 1 * time.Minute
	randomizationFactor  = 0.5
	multiplier           = 1.2
```